ethcore/src/lib.rs

```diff
@@ -114,7 +114,7 @@ extern crate heapsize;
 #[macro_use]
 extern crate ethcore_ipc as ipc;
 extern crate lru_cache;
-
+extern crate ethcore_stratum;
```

Comment:
nit: this and `lru_cache` should be in the section with the other non-`#[macro_use]` crates.